pkgs/by-name/ka/kak-tree-sitter-unwrapped/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromSourcehut,
+  nix-update-script,
+  testers,
+  kak-tree-sitter-unwrapped,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "kak-tree-sitter-unwrapped";
```

Comment:
This is a matter of taste, but
What about making this the default `kak-tree-sitter` while the wrapped version is called `kak-tree-sitter-wrapped`?

Reply:
> What about making this the default `kak-tree-sitter` while the wrapped version is called `kak-tree-sitter-wrapped`?

I would not like that, to be honest. My rationale here is that the package should have maximum functionality by default. You install it and you are done. 

If you really cannot afford the 16Mib necessary for `tcc` you can always get rid of it, but that does not justify imposing a worse out-of-the-box experience for other users.

Reply:
On the other hand the unwrapped version is more composable.
What if the programming environment I am using provides a C compiler upfront?
In this scenario tcc would be redundant.

But as I said it is a matter of taste, since both derivations are still available.